### ISSUE TITLE
Agent Ping $push fails when `ping` is not installed

### ISSUE THREAD

**Issue by jmalobicky** [medplum/medplum#4007]
Running the Agent in the `debain:bullseye-slim` image, `ping` is not installed by default. It would be good to check if `ping` is installed before issuing the `$push` - https://github.com/medplum/medplum/blob/v3.0.3/packages/agent/src/app.ts#L284

This leads to an inconsistent error message on the UI
```
Error
"$push" operation timed out. Agent may be unreachable
```

Though the Agent is connected and received the event. Below is the Agent log when trying to ping the docker gateway.
```
{"level":"INFO","timestamp":"2024-02-22T14:05:53.826Z","msg":"Medplum service started successfully"}
{"level":"ERROR","timestamp":"2024-02-22T14:09:48.075Z","msg":"Error during ping attempt to 172.17.0.1: Command failed: ping -c 4 172.17.0.1\n/bin/sh: 1: ping: not found\n"}
{"level":"ERROR","timestamp":"2024-02-22T14:09:48.076Z","msg":"Unknown message type: agent:error"}
```

**Comment by ThatOneBro** [comment]
Fixed by #4008

**Comment by jmalobicky** [comment]
> Fixed by #4008

@ThatOneBro THe commit I made that is referenced #4008 does in fact resolve the issue when running in a container. I do believe the error handling on the UI and checking if the `ping` utility even exists - should still be handled properly. If necessary, I can gladly open separate issues for the app error and a feature to check the the tools exist before execution?

**Comment by ThatOneBro** [comment]
@jmalobicky My mistake! I guess I jumped the gun on this one. You're correct, there should still be a check for the existence of the util and a corresponding UI error. I'll reopen this
